src/api/queries/queriesHelper.js

```diff
@@ -0,0 +1,19 @@
+function slugParam (slug) {
+  return `slug: "${slug}"`
+}
+
+function fromParam (from) {
+  return `from: "${toUTC_(from)}"`
+}
+
+function toParam (to) {
+  return `to: "${toUTC_(to)}"`
+}
+
+function intervalParam (interval = '1d') {
+  return `interval: "${interval}"`
+}
+
+function commonParams (slug, from, to, interval = '1d') {
```

Comment:
slugTimeRangeParams?

Reply:
They're the most common combination, hence the name. If you insist, I'll change that

Reply:
`common` does not carry any meaning at all, so it's not a fit name for a function. It's in the category of `foo` and `bar` imo